bundles/org.openhab.binding.teleinfo/src/main/resources/ESH-INF/thing/common-cbemm-channel-types.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="teleinfo"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="imaxType">
+		<item-type>Number:ElectricCurrent</item-type>
+		<label>IMAX</label>
+		<description>IMAX</description>
+		<category>Energy</category>
+		<state pattern="%d A" readOnly="true" />
+	</channel-type>
+
+	<channel-type id="iinstType">
+		<item-type>Number:ElectricCurrent</item-type>
+		<label>IINST</label>
+		<description>IINST</description>
+		<category>Energy</category>
+		<state pattern="%d A" readOnly="true" />
+	</channel-type>
+
+	<channel-type id="currentPowerType">
+		<item-type>Number:Power</item-type>
+		<label>currentpower</label>
+		<description>Instantaneous power</description>
+		<category>Energy</category>
+		<state pattern="%d W" readOnly="true" />
```

Comment:
Did you test what happens if the user displays the value in kW?
```suggestion
		<state pattern="%d %unit%" readOnly="true" />
```

Reply:
@fwolter Ok for your suggestion, but in this case, all updateState() operations shall be updated to set the unit data, no ?
E.g: updateState(CHANNEL_CBEMM_ISOUSC, new DecimalType(frame.getIsousc()));

Reply:
Yes, instead of DecimalType, QuantityType must be used.